ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -425,6 +434,12 @@ export default class TestDataModal extends React.Component {
           this.setState({ platform }, this.processOptions),
         title: 'Platform',
       },
+      {
+        options: tagOptions,
+        selectedItem: tag,
```

Comment:
Previous modal options have hardcoded a default value for their `selectedItem`. Our `tag` is initially hardcoded to an empty string, which isn't that useful. Rather set it here to the 1st item from `tagOptions`, if there are any.